### PR TITLE
feat(DAppsCopyUriInstructions): implemented popup with the instructions how to copy URI of DApps into Status app.

### LINKED ISSUE #14677: [UI - dapps Connection]  How to copy URL info dialog
[Design](https://www.figma.com/file/HrmZp1y4S77QJezRFRl6ku/Descoped-dApp-Interactions?type=design&node-id=3649%3A30334&mode=design&t=LpX0Wx80R6SXcYAl-1)

Screenshot 2024-05-09 at 20 04 08

Details of the `User Story` and scenarios to cover [HERE](https://www.notion.so/How-to-copy-the-dApp-URI-e884219a988a418ab6a7f0f5aa38e0f4)

Create the following information dialog that shall be shown when clicking on the `How to copy the dApp URL` link from main `Connect a dApp` dialog.

### PR DESCRIPTION
**Pull request by Seitseman**
Closes #14677

### Affected areas

image

### StatusQ checklist

- [x] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [x] test changes in both light and dark theme?

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

### CODE REVIEW

File: storybook/pages/DAppsWorkflowPage.qml [hunk 1]
```diff
@@ -25,12 +25,19 @@ import SortFilterProxyModel 0.2
...
 
-import utils 1.0
+import mainui 1.0
 import shared.stores 1.0
+import utils 1.0
 
 Item {
     id: root
 
+    Popups {
```
Comment (stefandunca) [inline storybook/pages/DAppsWorkflowPage.qml:35]: Could you please clarify why is this required, not easy to tell given that there is no reference to it.

File: ui/imports/shared/popups/qmldir [hunk 2]
```diff
@@ -32,4 +32,5 @@ SettingsDirtyToastMessage 1.0 SettingsDirtyToastMessage.qml
 UnblockContactConfirmationDialog 1.0 UnblockContactConfirmationDialog.qml
 UserAgreementPopup 1.0 UserAgreementPopup.qml
 UserStatusContextMenu 1.0 UserStatusContextMenu.qml
-ImageContextMenu 1.0 ImageContextMenu.qml
\ No newline at end of file
+ImageContextMenu 1.0 ImageContextMenu.qml
+DAppsUriCopyInstructionsPopup 1.0 DAppsUriCopyInstructionsPopup.qml
```
[the remark this answers is not in the document]
Comment (stefandunca) [reply ui/imports/shared/popups/qmldir:36]: It also looks to me that this is best kept in `ui/app/AppLayouts/Wallet/services/dapps` close to its context

**Review by stefandunca — APPROVED** [review]
Great move towards completing the connection user-flow.

File: storybook/pages/DAppsWorkflowPage.qml (hunk 1 above)
Comment (Seitseman) [reply storybook/pages/DAppsWorkflowPage.qml:35]: This is needed to make that "Instructions" popup to open from storybook. Another way would be to explicitly call "open" but then code for app and storybook would further diverge.
I will add a comment to clarify

File: ui/imports/shared/popups/qmldir (hunk 2 above)
[the remark this answers is not in the document]
Comment (Seitseman) [reply ui/imports/shared/popups/qmldir:36]: I would put it next to ui\imports\shared\popups\walletconnect\PairWCModal.qml as its also a StatusDialog-derived and is the only user of DAppsUriCopyInstructionsPopup

File: ui/imports/shared/popups/DAppsUriCopyInstructionsPopup.qml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+import StatusQ.Core.Theme 0.1
+import StatusQ.Popups 0.1
+import StatusQ.Popups.Dialog 0.1
+
+import utils 1.0
+import shared.controls 1.0
+
+StatusDialog {
+    width: 440
+    implicitHeight: 348
```
[the remark this answers is not in the document]
Comment (Seitseman) [reply ui/imports/shared/popups/DAppsUriCopyInstructionsPopup.qml:14]: To be fixed

**Review by alexjba — APPROVED** [review]
LGTM! 👍

File: storybook/pages/DAppsUriCopyInstructionsPage.qml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+    Button {
+        anchors.centerIn: parent
+        text: "Reopen"
+
+        onClicked: dialog.open()
+    }
+
+    DAppsUriCopyInstructionsPopup {
+        id: dialog
```
[the remark this answers is not in the document]
Comment (Seitseman) [reply storybook/pages/DAppsUriCopyInstructionsPage.qml]: to be done
